src/Entity/Coalition/CauseFollower.php

```diff
@@ -24,10 +35,56 @@ class CauseFollower extends AbstractFollower
      *
      * @ORM\ManyToOne(targetEntity="App\Entity\Coalition\Cause", inversedBy="followers")
      * @ORM\JoinColumn(onDelete="CASCADE", nullable=false)
+     *
+     * @Assert\NotNull(groups={"write"})
      */
     private $cause;
 
-    public function __construct(Cause $cause, Adherent $adherent)
+    /**
+     * @ORM\Column(length=50, nullable=true)
+     * @Assert\NotBlank(groups={"write"})
+     * @Assert\Length(max=50, groups={"write"})
+     */
+    private $firstName;
+
+    /**
+     * @ORM\Column(nullable=true)
+     *
+     * @Assert\NotBlank(groups={"write"})
+     * @Assert\Email(message="common.email.invalid", groups={"write"})
+     * @Assert\Length(max=255, maxMessage="common.email.max_length", groups={"write"})
+     */
+    private $emailAddress;
+
+    /**
+     * @var Zone|null
+     *
+     * @ORM\ManyToOne(targetEntity="App\Entity\Geo\Zone")
+     * @ORM\JoinColumn(onDelete="SET NULL")
+     *
+     * @Assert\NotNull(groups={"write"})
+     */
+    private $zone;
+
+    /**
+     * @ORM\Column(type="boolean", nullable=true)
+     *
+     * @Assert\NotNull(groups={"write"})
+     * @Assert\IsTrue(groups={"write"})
+     */
+    private $cguAccepted;
+
+    /**
+     * @ORM\Column(type="boolean", nullable=true)
```

Comment:
what about setting a default `false` value, it will simpler to do with two values: `true/false` instead of `null/true/false`, wdyt?